clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java

```diff
@@ -1591,7 +1597,7 @@ protected void processCommonConsumerAction(
         storageUtilizationManager.initPartition(partition);
         break;
       case UNSUBSCRIBE:
-        LOGGER.info("{} UnSubscribed to: {}", consumerTaskId, topicPartition);
+        LOGGER.info("{} Unsubscribing to: {}", consumerTaskId, topicPartition);
```

Comment:
nit: may be remove this line completely ?

Reply:
No, I don't want since the following unsubcribing logic could fail, such as buffer drain could timeout.
I left this intentionally.